### DISCUSSION TITLE
Vimeo video is getting stuck in app while play properly in mobile browser

### DISCUSSION THREAD

**Discussion by RohitStartbit** [vimeo/player.js#1006] [Bug reports]
We have one react app, in this we have integrated vimeo videos through Web view + iframe. In mobile app - androind version, video is not playing properly and getting stuck after each 2 or 3 seconds, while when we play same video on Chrome - mobile browser on same tablet and mobile, it plays well.

What could be the possible issue?

**Comment by luwes** [comment]
@RohitStartbit thanks for the feedback. 
do you have a test page online? is it a specific video? that would help to debug.

**Comment by RohitStartbit** [comment]
Hi,
Not a specific videos, all the videos in the app in specific mobiles are giving problem, while we play same video in mobile browser, it works well. We have used webview and iframe.

**Comment by luwes** [comment]
what is loaded in the webview? just the Vimeo player?

**Comment by RohitStartbit** [comment]
In webview, we are loading iframe and in iframe vimeo video is loaded.

**Comment by jhso-dev** [comment]
Hi @luwes ,
Same issue here
More specifically, it happens when two or more vimeo players are played on a web page in **Android Webview** but everything works ok in in-app browser.
I'd like to share this video https://drive.google.com/file/d/1aAZMmQrqMVEi9XS7kZ1WkDfEYZP5XgIK/view?usp=sharing
plus If you don't mind, can you let me know the latest update?
I think It worked ok a month ago.

Thanks in advance!

**Comment by qleeyun** [comment]
@luwes Same issue

**Comment by heyman333** [comment]
+1

**Comment by luwes** [comment]
is this perhaps related to this issue #298?
solution https://stackoverflow.com/questions/38277526/webview-content-security-policy/40412110#40412110

are there any errors showing in the debugger?
network requests being blocked?

**Comment by jhso-dev** [comment]
Hi @luwes,
It's not related to the issue #298, and network ain't problem.
Please check this video https://drive.google.com/file/d/1FI3Z0TMxwHT1-e3EyBzMjdg3EEMxNkHV/view?usp=sharing
And check this webpage https://collection-87ba6.firebaseapp.com/

YouTube and Dailymotion players worked in the same test.

Thanks in advance!

**Comment by RohitStartbit** [comment]
Any one has any suggestion OR solution?
We got reply from vimeo support team that we have to use third party player with external URL of vimeo video, but still on few devices mostly Android version 10, we are facing problem of lip syncing of the video and audio is not matching...

**Comment by luwes** [comment]
do you have any Android code to share? how do you create the webview, with which options, etc?

https://www.google.com/search?q=android%20webview%20vimeo

is it a WebChromeClient?

**Comment by jhso-dev** [comment]
@luwes Sorry for the late.
We are using [this webview](https://github.com/react-native-community/react-native-webview).
It is WebChromeClient. 
Thanks in advance!

**Comment by jhso-dev** [comment]
@luwes Hi luwes any updates?

**Comment by luwes** [comment]
have you tried setting the `autopause` parameter to `0`. since it happens with multiple players that could be a potential cause. that feature basically pauses the other players on the page when starting a new one but maybe there is something weird going on. not sure.

**Comment by jimbolo** [comment]
I have a similar issue Facebook in App Webview, Neither Vimeo nor YouTube plays. Here is the screencast which shows the issue,  I am clicking on a link using Facebook mobile app and it takes me Facebook in app webview: https://drive.google.com/file/d/11Vr1vUml-9X4FbXH-bktILG-iwwTQS1e/view?usp=sharing . 
Please visit our Facebook page https://www.facebook.com/salsamuzik/ with your Facebook app and find the post which has this url http://salsalar.com/station/salsita-mix-vol-02-dj-yedi/ you should be able to troubleshoot this way (https://www.facebook.com/salsamuzik/posts/112510477546891)
